doc/whats_new/v1.3.rst

```diff
@@ -135,6 +135,11 @@ Changelog
   out-of-bag scores via the `oob_scores_` or `oob_score_` attributes.
   :pr:`24882` by :user:`Ashwin Mathur <awinml>`.
 
+- |Feature| :class:`ensemble.HistGradientBoostingRegressor` and
+  :class:`ensemble.HistGradientBoostingClassifier` treats negative values
+  as missing values for categorical features, following LightGBM's convention.
+  :pr:`25629` by `Thomas Fan`_.
```

Comment:
I think we should consider this a bugfix, because of:
- the lack of consistency between of fit and predict that was reported by OP as a bug,
- the fact our docstring said that integers where expected to be positive only, even if we did not do the proper input validation at fit time,
- we don't want to implement a deprecation cycle for this behavioral change ;)

Here is a suggestion for the phrasing:

```suggestion
- |Fix| :class:`ensemble.HistGradientBoostingRegressor` and
  :class:`ensemble.HistGradientBoostingClassifier` treats negative values for
  categorical features consistently as missing values, following LightGBM's and
  pandas' conventions.
  :pr:`25629` by `Thomas Fan`_.
```

However this entry needs to be moved below to be after the "Enhancement" entries. I cannot do that via the github UI.

Reply:
Yea, a bug fix makes sense here. I update the title and updated+moved the change log entry.